config_hub/finetune/tiny-llama/qlora.yaml

```diff
@@ -85,18 +85,6 @@ train:
   # Whether to tie the embedding weights with the language modeling head weights. (type: Optional[bool], default: null)
   tie_embeddings:
 
-  #   (type: float, default: 0.0003)
-  learning_rate: 0.0002
-
-  #   (type: float, default: 0.02)
-  weight_decay: 0.0
-
-  #   (type: float, default: 0.9)
-  beta1: 0.9
-
-  #   (type: float, default: 0.95)
-  beta2: 0.95
-
```

Comment:
I guess we don't care about backwards compatibility since the other CLI changes related to the checkpoint dir need to be done

Reply:
Yeah, I think full backwards compatibility would probably also add so much complexity and overhead. I think we need to wrap all the CLI changes as soon as possible, and then we hopefully never have to do a breaking change like this again.